### PR TITLE
feat: syncfolder example

### PR DESCRIPTION
**Pull request by allanbowe**
This PR provides a sample syncFolder, of the type that could be implemented as part of the work to deliver this feature in the sasjs/cli:  https://github.com/sasjs/cli/issues/1209

### CODE REVIEW

File: sasjs/sasjsconfig.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
       "sasjs/services/common"
     ]
   },
+  "syncFolder": "mocks",
```
[the remark this answers is not in the document]
Comment (allanbowe) [reply sasjs/sasjsconfig.json:11]: aah yes, agreed, will update!

File: sasjs/mocks/services/common/appinit.js [hunk 2]
```diff
@@ -0,0 +1 @@
+// this file will return a response
```
Comment (YuryShkoda) [inline sasjs/mocks/services/common/appinit.js:1]: please add blank line

File: sasjs/mocks/services/common/getdata.js [hunk 3]
```diff
@@ -0,0 +1 @@
+// this file will take input and return a response
```
Comment (YuryShkoda) [inline sasjs/mocks/services/common/getdata.js:1]: please add blank line

File: package-lock.json [hunk 4]
```diff
@@ -77,6 +77,12 @@
         "@types/node": "*"
       }
     },
+    "node_modules/@types/tough-cookie": {
```
Comment (YuryShkoda) [inline package-lock.json:80]: Why `package-lock.json` has been changed without changes to `package.json`?

**Review by YuryShkoda — CHANGES_REQUESTED** [review]

**Pull request merged**